grapple/types/pages.py

```diff
@@ -80,7 +80,7 @@ def resolve_parent(self, info, **kwargs):
         Docs: https://docs.wagtail.io/en/stable/reference/pages/model_reference.html#wagtail.core.models.Page.get_parent
         """
         try:
-            return self.get_parent().specific
+            return self.get_parent().live().public().specific
```

Comment:
`self.get_parent()` returns a `Page` and `Page` doesn't have the `.live()` and `.public()` methods.

Reply:
True, I missed that. Actually in that situation it makes sense to not check for live and public.

Reply:
Nice. Now just fix the linting issues and will be good to go.